test/e2e_node/allocatable_eviction_test.go

```diff
@@ -22,9 +22,9 @@ import (
 
 	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
-	nodeutil "k8s.io/kubernetes/pkg/util/node"
 	"k8s.io/kubernetes/pkg/kubelet/apis/kubeletconfig"
 	"k8s.io/kubernetes/pkg/kubelet/cm"
+	nodeutil "k8s.io/kubernetes/pkg/util/node"
```

Comment:
Same as above. Any reasons to change the import? Seems no need to modify this file.